crates/sui-swarm/src/memory/swarm.rs

```diff
@@ -345,6 +357,19 @@ impl Swarm {
     pub fn fullnodes(&self) -> impl Iterator<Item = &Node> {
         self.fullnodes.values()
     }
+
+    pub async fn spawn_new_fullnode(&mut self, config: NodeConfig) -> SuiNodeHandle {
+        let name = config.protocol_public_key();
+        let node = Node::new(config);
```

Comment:
it would be great if we could add something to the name of the fullnode nodes so that they can be distinguished from validators in the logs